SQL/DDL.pm

```diff
@@ -1028,6 +1027,21 @@ CREATE INDEX user_default_subscription_t5 ON user_default_subscription_t (
   user_id
 )
 /
+
+--
+-- login_attempt_t
+--
+ALTER TABLE login_attempt_t
+  ADD CONSTRAINT login_attempt_t2
+  FOREIGN KEY (realm_id)
+  REFERENCES realm_owner_t(realm_id)
+/
+CREATE INDEX login_attempt_t3 ON login_attempt_t (
+  realm_id,
+  creation_date_time,
+  login_attempt_id
```

Comment:
Don't add login_attempt_id, because it complicates the index. There's already an index on that.